upload_about_me.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+require 'functions.php';
+
+$db = displayPDO();
+
+$stmt = $db->prepare (
+                "UPDATE `about_me`
+                 SET `paragraph_1` = :paragraph_1, `image` = :image, `paragraph_2` = :paragraph_2
+                 WHERE id = 1;"
+);
+
+$stmt->bindParam(':paragraph_1', $_POST["project_title"]);
```

Comment:
no validation on if project_title actually exists inside the POST array

Reply:
complete.